code/go/0chain.net/smartcontract/partitions/partitions.go

```diff
@@ -803,3 +803,43 @@ func (p *Partitions) Msgsize() int {
 }
 
 type partitionsDecode Partitions
+
+func (p *Partitions) RepairValidatorPartitions(balances state.StateContextI) error {
```

Comment:
can we make it generic for all the partitions? we can pass the name of partitions and load from MPT by name